libs/eslint-plugin-vx/src/rules/index.ts

```diff
@@ -63,6 +61,6 @@ const rules: Record<
   'no-assert-truthiness': noAssertStringOrNumber,
   'no-floating-results': noFloatingVoids,
   'no-import-workspace-subfolders': noImportSubfolders,
-};
+} as unknown as Record<string, Rule.RuleModule>;
```

Comment:
This waters down the specificity of the types to just be standard ESLint `RuleModule`s rather than TSESLint's `RuleModule`. Consumers of this library don't need to know they're built using `@typescript-eslint/experimental-utils`.